components/Sections/MyStudio.tsx

```
@@ -1,41 +1,58 @@
+  return (
+    <section 
+      className={`py-16 bg-gray-50 ${className}`}
+      aria-labelledby="studio-heading"
+      role="region"
+    >
+      <div className="container mx-auto px-4 max-w-4xl">
+        <h2 id="studio-heading" className="text-3xl font-bold mb-8">
+          {STUDIO_CONTENT.title}
+        </h2>
+        <div className="flex flex-col lg:flex-row gap-6 md:gap-8 lg:gap-12 items-center">
+          <div className="lg:w-2/3" role="main">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<main>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)